### ISSUE TITLE
NTR: Methylmalonic semialdehyde dehydrogenase deficiency (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8200]
New term request Methylmalonic semialdehyde dehydrogenase deficiency (segal_071107213835):

**Comment by pnrobinson** [comment]
@MickeySegal
Note I think this enzyme is measured in fibroblasts, so I altered the term as below

Adding new term: 
- ID: HP:6000219
- label: Reduced methylmalonate semialdehyde dehydrogenase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195) | [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:23835272
- def: The activity of methylmalonate semialdehyde dehydrogenase ( EC 1.2.1.27) in cultured fibroblasts is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8200
- synonyms: Methylmalonic semialdehyde dehydrogenase deficiency(NONE)
